nibabel/tests/test_proxy_api.py

```diff
@@ -216,11 +221,10 @@ def obj_params(self):
             offsets = (self.header_class().get_data_offset(),)
         else:
             offsets = (0, 16)
-        slopes = (1., 2.) if self.has_slope else (1.,)
-        inters = (0., 10.) if self.has_inter else (0.,)
-        dtypes = (np.uint8, np.int16, np.float32)
+        slopes = (1., 2., float(np.float32(3.1416))) if self.has_slope else (1.,)
+        inters = (0., 10., float(np.float32(2.7183))) if self.has_inter else (0.,)
```

Comment:
Stupidly picked decimals that couldn't be represented exactly. Cast them to float32 to make it concrete, then back to `float` for consistency.

Can pick other numbers if this will just lead to confusion.

Reply:
Maybe add this as a comment, otherwise makes sense with explanation